diff --git a/build/common.gypi b/build/common.gypi
index 7ec5ae2..ee2323e 100644
--- a/build/common.gypi
+++ b/build/common.gypi
@@ -2815,7 +2815,7 @@
           '_SCL_SECURE_NO_DEPRECATE',
           # This define is required to pull in the new Win8 interfaces from
           # system headers like ShObjIdl.h.
-          'NTDDI_VERSION=0x06020000',
+          'NTDDI_VERSION=0x06030000',
           # This is required for ATL to use XP-safe versions of its functions.
           '_USING_V110_SDK71_',
         ],
