diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
index 9ad44a9..23e68ff 100644
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,7 +1,7 @@
-COMPANY_FULLNAME=The Chromium Authors
-COMPANY_SHORTNAME=The Chromium Authors
+COMPANY_FULLNAME=The Iridium Authors
+COMPANY_SHORTNAME=The Iridium Authors
 PRODUCT_FULLNAME=Iridium
 PRODUCT_SHORTNAME=Iridium
 PRODUCT_INSTALLER_FULLNAME=Iridium Installer
 PRODUCT_INSTALLER_SHORTNAME=Iridium Installer
-COPYRIGHT=Copyright 2014 The Chromium Authors. All rights reserved.
+COPYRIGHT=Copyright 2015 The Iridium Authors. All rights reserved.
