diff --git a/third_party/ffmpeg/chromium/ffmpeg.sigs b/third_party/ffmpeg/chromium/ffmpeg.sigs
index e3fa653..e422803 100755
--- a/third_party/ffmpeg/chromium/ffmpeg.sigs
+++ b/third_party/ffmpeg/chromium/ffmpeg.sigs
@@ -75,3 +75,4 @@ int av_image_fill_linesizes(int *linesizes, enum PixelFormat pix_fmt, int width)
 AVBufferRef *av_buffer_create(uint8_t *data, int size, AVFreeOperation free, void *opaque, int flags);
 int avcodec_fill_audio_frame(AVFrame *frame, int nb_channels, enum AVSampleFormat sample_fmt, const uint8_t *buf, int buf_size, int align);
 void *av_buffer_get_opaque(const AVBufferRef *buf);
+void av_max_alloc(size_t max);
diff --git a/third_party/ffmpeg/ffmpeg.gyp b/third_party/ffmpeg/ffmpeg.gyp
index 07f229e..a2bef1a 100755
--- a/third_party/ffmpeg/ffmpeg.gyp
+++ b/third_party/ffmpeg/ffmpeg.gyp
@@ -451,6 +451,16 @@
                   'sources': [
                     '<(shared_generated_dir)/ffmpeg.def',
                   ],
+                  'conditions': [
+                    ['OS=="win" and win_use_allocator_shim==1', {
+                      'dependencies': [
+                        '../../base/allocator/allocator.gyp:allocator',
+                      ],
+                      'sources': [
+                        'chromium/dllmain.cc',
+                      ],
+                    }],
+                  ],
                   'actions': [
                     {
                       'action_name': 'generate_def',
